stdlib/public/core/Tuple.swift.gyb

```diff
@@ -111,7 +111,7 @@ public func >=(lhs: (), rhs: ()) -> Bool {
 %   equatableTypeParams = ", ".join(["{} : Equatable".format(c) for c in typeParams])
 
 %   originalTuple = "(\"a\", {})".format(", ".join(map(str, range(1, arity))))
-%   greaterTuple = "(\"a\", {})".format(", ".join(map(str, range(1, arity - 1) + [arity])))
+%   greaterTuple = "(\"a\", {})".format(", ".join(map(str, list(range(1, arity - 1)) + [arity])))
```

Comment:
Don't need this change, either.

Reply:
Python 3 _definitely_ went bananas without this change.

Reply:
I think this change is needed. In Python 3, `range` will return an iterable, which cannot be added a list. The `list(…)` transforms the iterable into a list, which will work.

Reply:
Yes, because if you enter `arity=3;  "(\"a\", {})".format(", ".join(map(str, range(1, arity - 1) + [arity])))` to an interpreter it will cause `TypeError: unsupported operand type(s) for +: 'range' and 'list'` also https://stackoverflow.com/a/47472763